client/src/app/pages/dashboard/profile/profile-run-history/profile-run-history.component.ts

```diff
@@ -46,10 +56,14 @@ export class ProfileRunHistoryComponent implements OnInit {
       this.loadRunHistory();
     });
   }
+
   loadRunHistory() {
     this.usersService.getRunHistory(this.user.id, {
       params: {
         expand: 'map',
+        mapName: this.filterFG.value.map,
+        isPB: this.filterFG.value.isPersonalBest,
+        order: this.filterFG.value.order,
```

Comment:
Shouldn't this be using `this.currentFilter` ?

Reply:
It might matter in edge cases that it isn't `this.currentFilter` (as in it might not work right as `this.filterFG.value` but even if it doesn't it makes more sense and is cleaner to make it so. Changed in 97e96fe